physionet-django/console/templates/console/training_process.html

```diff
@@ -48,9 +45,28 @@ <h5>Training</h5>
                   {% if training.completion_report_url %}
                     <li>URL: <mark><a href="{{ training.completion_report_url }}" target="_blank">External link</a></mark></li>
                   {% endif %}
+                  <li>Submitted: {{ training.application_datetime|date }}</li>
                 </ul>
             </div>
         </div>
+        <div class="card mb-4">
+            <div class="card-header">
+                Document information
+            </div>
+            <div class="card-body">
+                <p>
+                    {% for key, value in parsed_training_pdf.items %}
```

Comment:
```suggestion
                    {% for field_name, matched_value in parsed_training_pdf.items %}
```
Maybe a bit better? Not very much though